docs/devel/consumers-claims.md

```diff
@@ -0,0 +1,309 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/devel/consumers-claims.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# The consumer/claim model for cluster resources
+
+A pattern has emerged for managing certain classes of resources (e.g. storage
+and networking) in the Kubernetes API.  This doc aims to document this pattern
+for future reuse.
+
+**Table of Contents**
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+
+- [The consumer/claim model for cluster resources](#the-consumerclaim-model-for-cluster-resources)
+  - [Background](#background)
+  - [Status](#status)
+  - [General model](#general-model)
+  - [Storage](#storage)
+    - [PersistentVolumeClaim](#persistentvolumeclaim)
+    - [VolumeManagers](#volumemanagers)
+    - [Config](#config)
+    - [Scrubbing](#scrubbing)
+  - [Networking](#networking)
+    - [IngressClaim](#ingressclaim)
+    - [LoadBalancerManagers](#loadbalancermanagers)
+    - [Config](#config)
+    - [Scrubbing](#scrubbing)
+  - [Future](#future)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
+## Background
+
+Some types of resources which are used in a cluster follow a common lifecycle:
+they are provisioned (whether manually or automatically), used by some
+consumer(s), eventually released by the consumer, scrubbed, and finally either
+released or reused.  These resources generally map to discrete physical or
```

Comment:
deleted or reused?

"released" is indistinguishable from the previous use of that word in this sentence.